content/configuration/source.md

```diff
@@ -0,0 +1,333 @@
+---
```

Comment:
You can update also https://toggle-recommendations-docs.botkube-docs-dcb.pages.dev/configuration/communication/#source-and-executor-bindings

to link to this doc (same as we have for executors)
![Screen Shot 2022-08-12 at 11 08 35](https://user-images.githubusercontent.com/17568639/184322948-22c901f6-b903-447e-8ee4-a714d20905ee.png)